STEERING-COMMITTEE.md

```diff
@@ -1,74 +1,146 @@
 # Istio Steering Committee
 
 The Istio Steering Committee was formed to oversee the administrative aspects of the project, including governance, branding, and marketing.
-Steering was created to allow the Technical Oversight Committee to exclusively focus on the technical aspects of the project. 
+Steering was created to allow the Technical Oversight Committee to exclusively focus on the technical aspects of the project.
 
 * [Charter](#charter)
+* [Membership and Voting](#membership-and-voting)
 * [Committee meetings](#committee-meetings)
 * [Committee members](#committee-members)
 * [General questions](#general-questions)
 * [Getting in touch](#getting-in-touch)
 
 ## Charter
 
-1. Define, evolve, and defend the vision, values, mission, and scope of the project - to establish and maintain the soul of Istio.
-2. Define and evolve project governance structures and policies, including how contributors become committers/maintainers, approvers, reviewers, members, code of conduct, etc.
-3. Control access to, establish processes regarding, and provide a final escalation path for any Istio repository.
-4. Control and delegate access to and establish processes regarding other project resources and assets, including artifact repositories, build and test infrastructure, web sites and their domains, blogs, social-media accounts, etc.
-5. Manage the Istio brand to decide which things can be called “Istio” and how that mark can be used in relation to other efforts or vendors.
-6. Resolve any dispute from the Technical Oversight Committee.
+The Steering Committee’s responsibilities include:
```

Comment:
I believe the framework and oversight of an Istio conformance testing/program should be under the responsibilities of the SC as well.

Reply:
That will probably be true, and handled under "Advising the trademark owner on issues relating to the Istio trademark and logo."